### PR TITLE
267 added new json avro converter for smooth migration

### PR DESCRIPTION
**Pull request by wojtkiewicz**
This PR contains integration with new json-avro-converter. Library has not been opensourced yet but hopefully will be.

### CODE REVIEW

**Comment by druminski** [comment]
:+1:

File: hermes-common/build.gradle [hunk 1]
```diff
@@ -23,15 +23,9 @@ dependencies {
...
-     *
-     * https://issues.apache.org/jira/browse/AVRO-1582
-     * https://issues.apache.org/jira/browse/AVRO-1717
-     *
-     * They improve conversion of json to avro.
-     * Patched version will be used until mentioned fixes are released in official version of avro.
+     * This is an early adoption of json to avro converting library that has not been opensourced and published yet.
      */
-    compile files('./lib/avro-1.8.0-json2avroPatch-SNAPSHOT.jar')
+    compile files('./lib/json-avro-converter-0.1.1.jar')
```
Comment (pawelszymczyk) [inline hermes-common/build.gradle]: I hope it's temporary solution

**Comment by pawelszymczyk** [comment]
:+1:

File: hermes-common/build.gradle (hunk 1 above)
Comment (wojtkiewicz) [inline hermes-common/build.gradle]: absolutely

**Pull request merged**